README.md

```diff
@@ -1,11 +1,11 @@
 # Editor SDK Integration Examples
 
 In this repository you can find some examples on integrating the editor-sdk.
-We strongly advice to always run your code in a local webserver.
+We strongly advice to always run your code in a local webserver to ensure all funcations are available.
```

Comment:
functions

Reply:
Done in https://github.com/chili-publish/editor-sdk-integration-examples/pull/3/commits/a6b4a00f0e82c5e1d31dbd2e59e14b134390032b